composer.json

```diff
@@ -20,7 +20,6 @@
         "incenteev/composer-parameter-handler": "~2.0",
         "friendsofsymfony/user-bundle": "~1.3",
         "doctrine/doctrine-fixtures-bundle": "2.2.*",
-        "knplabs/knp-paginator-bundle": "~2.4",
```

Comment:
This will break the current usage of the bundle. You have to replace the use of that paginator with the one you're introducing. Also, if you change this file the `composer.lock` has to be updated.